storage/dvo_recommendations_storage.go

```diff
@@ -170,3 +171,12 @@ func (storage DVORecommendationsDBStorage) MigrateToLatest() error {
 		storage.GetMigrations(),
 	)
 }
+
+// ReportsCount reads number of all records stored in the dvo.dvo_report table
+func (storage DVORecommendationsDBStorage) ReportsCount() (int, error) {
```

Comment:
It returns -1 if no records were found + OCP, Redis and Noop storages use `int`. I'm not saying `uint` is not better for this case, but it would mean changing other 3 structs + methods 😓